modules/custom-status/rest/custom-status-endpoint.php

```diff
@@ -329,6 +337,19 @@ public static function handle_reorder_status( WP_REST_Request $request ) {
 		return rest_ensure_response( $custom_status_module->get_custom_statuses() );
 	}
 
+	// Utility functions
+
+	private static function is_valid_user_ids( $user_ids ) {
+		foreach ( $user_ids as $user_id ) {
+			$user = get_user_by( 'id', $user_id );
```

Comment:
If we use [this](https://stackoverflow.com/a/62574404), this can be done in one call so it'd be a bit more efficient and faster.

Reply:
Good thinking! I can update that in another PR.